widget/entry.go

```diff
@@ -102,16 +95,14 @@ type Entry struct {
 	// undoStack stores the data necessary for undo/redo functionality
 	// See entryUndoStack for implementation details.
 	undoStack entryUndoStack
-
-	// doubleTappedAtUnixMillis stores the time the entry was last DoubleTapped
-	// used for deciding whether the next MouseDown/TouchDown is a triple-tap or not
-	doubleTappedAtUnixMillis int64
 }
 
 // NewEntry creates a new single line entry widget.
 func NewEntry() *Entry {
 	e := &Entry{Wrapping: fyne.TextWrap(fyne.TextTruncateClip)}
 	e.ExtendBaseWidget(e)
+
+	e.syncSelectable()
```

Comment:
I don't think this should be here. If it is needed here then we can't guarantee that someone using the widget directly from the struct get consistent result. Better to just let the CreateRenderer call sync stuff like we do with the test of the providers.